packages/bcp/package.json

```diff
@@ -34,7 +34,7 @@
     "build-or-skip": "[ -n \"$SKIP_BUILD\" ] || yarn build",
     "test-node": "node jasmine-testrunner.js",
     "test-firefox": "yarn pack-web && karma start --single-run --browsers Firefox",
-    "test-chrome": "yarn pack-web && karma start --single-run --browsers ChromeHeadlessInsecure",
+    "test-chrome": "yarn pack-web && karma start --single-run --browsers ChromeHeadless",
```

Comment:
Always nice to see the word `Insecure` disappear from the codebase :smile:

Reply:
Yeah this was never needed here. Will created that for IOV because of CORS issues in the original Gaia docker container.